_posts/2017-08-18-gsoc_2017-cynexit.markdown

```diff
@@ -1,19 +1,72 @@
 ---
-title: Project Title
-subtitle: Project Subtitle
+title: Holmes-Analytics
+subtitle: Interface, Scheduler & Scalable Framework
 layout: default
 modal-id: 1
 category: gsoc2017
 date: 2017-01-01
-img: temp.svg
-thumbnail: temp.svg
+img: investigation.svg
+thumbnail: investigation.svg
 alt: license-img
 repository: https://github.com/HolmesProcessing/gsoc_
 documentation: 
 license: 
 
 ---
 
-##### Overview
 
-Content Goes Here
+### Preamble
+Back in the days when Holmes-Processing was born it only consisted of one tool, Totem. Totem excels at feature extraction and is still one of the most flexible and streamlined tools for this job on the market. After playing with Totem for a while it was obvious that we needed to get away from the single threaded python script that acted as a webserver for Totem for pull files from and also away from simply dumping results into mongoDB.
+
+To tackle both issues we needed a general-purpose storage tool that would abstract all of that away by providing a simple API to retrieve and save files as well as data from multiple backends. This led to the development of Holmes-Storage to make it easier for admins and researchers alike to work with Totem and its results since you could just switch from mongoDB to Cassandra and from files on a local disk to S3 or Minio without touching Totem or anything attached to the rest of the infrastructure.
+
+A few other smaller tools followed to make Holmes-Processing more feasible for "everyday use" like Gateway and a simple frontend (that quickly got outdated), but one big piece was always missing: a native way to handle and schedule our own machine learning tasks based on the data we extracted. This is the reason Holmes-Analytics was developed.
+
+
+### The Project
+Holmes-Analytics is a scheduler for various data-driven tasks like machine learning and advanced statistics as well as a scalable framework for these kinds of jobs - or services if you will - to easily integrate them into the Holmes-Processing ecosystem and connect them with your data without the need for overly specific backbone handling.
+
+
+### Technical Deep Dive
+Holmes-Analytics is written in Scala using the actor model provided by akka as much as possible. If you are not familiar with this concept you can check out this good and short introduction from Brian Storti.
```

Comment:
using, as much as possible, the actor model provided by akka.
add a link to Brian Storti